pkg/kubernetes/osiris_test.go

```diff
@@ -65,7 +65,7 @@ func TestResourceIsOsirisEnabled(t *testing.T) {
 			actual := ResourceIsOsirisEnabled(test.annotations)
 			if actual != test.expectedResult {
 				t.Errorf(
-					"expected GetMinReplicas to return %t, but got %t",
+					"expected ResourceIsOsirisEnabled to return %t, but got %t",
```

Comment:
This is good for now. I think that we should shortly look to start using testify for making assertions and these kind of messages that provide an English description of the failed assertion, which are frankly a pain to write, become optional.